source/index.rst

```diff
@@ -20,3 +31,107 @@ Indices and tables
 * :ref:`modindex`
 * :ref:`search`
 
+
+Structure of Guide
+==================
+
+Introduction
+  - What is Ethereum?
+  - The Homestead Release
+  - History of Ethereum[focus on brevity vs completeness with links to outside resources]
+  - Disclaimer [security/legal warning, etc.]
+  - Glossary [of Common Ethereum Terms]
+
+Ethereum Ecosystem
+  - Ethereum Network Stats
+  - Client Implementations
+  - EVM Compilers
+  - Block Explorers
+  - Miscellaneous Tools
+  - Exchanges
+  - Mining Pools
+  - Base Layer Services
+  - DApps
+  - Community
+    * Fora [github, wiki, gitter, reddit, forum, stackoverflow]
+    * Hubs
+    * Meetups
+
+Using Ethereum: The Basics
+  - Installing a Client
+    * Linux
+    * Windows
+    * Mac
+    * iOs
+    * Android
+    * Raspberry Pi
+    * ARM
+    * Binaries
+    * Source Installation
+  - Creating an Account
+    * Using Console
+    * Using EthKey
+    * Using Mist
+    * Using RPC
+  - Account Management
+    * What are keys, wallets, and accounts?
+    * Keys
+    * Wallets
+    * Accounts (EOA) [Externally Owned Accounts are referenced here. make sure to differentiate from contract accounts that are mentioned later in the guide]
+  - Backup/Import Accounts
+  	* Backing Up Wallets
+	* Importing Wallets [including pre-sale wallets]
+	* Online wallets, Paper Wallets and Cold Storage
+  - Ether
+    * What is Ether?
+    * [Table of Sub Units]
+    * Getting Ether [Will point to mining and exchanges pages with explanation of both]
+    * Sending Ether
+  - Mining
```

Comment:
so here we need a section on running a node, syncing, fast syncing and light client
network connectivity , NAT, discovery and peers